automated install/basic-install.sh

```diff
@@ -2207,8 +2207,23 @@ FTLinstall() {
     # Move into the temp ftl directory
     pushd "$(mktemp -d)" > /dev/null || { printf "Unable to make temporary directory for FTL binary download\\n"; return 1; }
 
-    # Always replace pihole-FTL.service
-    install -T -m 0755 "${PI_HOLE_LOCAL_REPO}/advanced/Templates/pihole-FTL.service" "/etc/init.d/pihole-FTL"
+    # Ensure /opt/pihole exists, then install fix_files.sh
+    install -T -D -m 0755 "${PI_HOLE_LOCAL_REPO}/advanced/fix_files.sh" "${PI_HOLE_INSTALL_DIR}/fix_files.sh"
+    # Always replace pihole-FTL
+    install -T -m 0755 "${PI_HOLE_LOCAL_REPO}/advanced/Templates/pihole-FTL.sh" "/etc/init.d/pihole-FTL"
+    
+    if is_command systemctl ; then
+        local systemd_unit_file="/etc/systemd/system/pihole-FTL.service"
+        if [[ ! -f "${systemd_unit_file}" && -d "${systemd_unit_file}.d" ]]; then
```

Comment:
I would not error out here since there are plenty of other reasons to use override files, e.g. service hardening or nice/priority levels, Before=/After= ordering and all such. But a warning would be fine instead, so users are reminded to check back if anything fails.

Reply:
That check exists for cases where override files were created to work with the generated system-v units, so it would error out to prevent unintended effects from combining those with a new systemd definition. It might still be a good idea to notify the user if the systemd file is updated and existing overrides exist.

Reply:
Yes I understood the reason for this check, but as said, it breaks the install when there are non-evil override files for several other good reasons. IMO it would be better to assume it's non-harmful and warn/remind users to check it in case of errors, instead of forcing all to move all overrides out of place, run the install, move overrides into place again, restart service... and probably then the installer fails because for some special permissions/capabilities on `pihole-FTL.service` the override is required.